components/post_view/post_body/index.ts

```diff
@@ -3,13 +3,28 @@
 
 import {connect} from 'react-redux';
 import {getPost} from 'mattermost-redux/selectors/entities/posts';
-import {isCurrentChannelReadOnly, getCurrentChannel} from 'mattermost-redux/selectors/entities/channels';
+import {
+    isCurrentChannelReadOnly,
+    getCurrentChannel,
+} from 'mattermost-redux/selectors/entities/channels';
 import {getUser} from 'mattermost-redux/selectors/entities/users';
 import {getConfig} from 'mattermost-redux/selectors/entities/general';
 
-import PostBody from './post_body.jsx';
+import {Post} from 'mattermost-redux/types/posts';
+import {GlobalState} from 'mattermost-redux/types/store';
 
-function mapStateToProps(state, ownProps) {
+import {PluginsState} from 'types/store/plugins';
+
+import PostBody from './post_body';
+
+interface State extends GlobalState {
```

Comment:
We have actually have a version of `GlobalState` declared in the webapp that includes this, so you don't need this interface.
You can import it from `types/store`.

Reply:
I have imported from `import {GlobalState} from 'mattermost-redux/types/store';` here only. Are you suggesting something else?

Reply:
Yes, you should use the type provided by `types/store`:

`import {GlobalState} from 'types/store'`

Reply:
I was getting this error thats why I declared the type `State` @devinbinnie 


![image](https://user-images.githubusercontent.com/351564/101331201-a687c700-3899-11eb-861d-4c97b394b7fc.png)

Reply:
Is your branch synced with `master`? We added the `plugins` field quite recently to that type AFAIK.

Reply:
I tried to sync the master... even in the `store.d.ts` there is no `plugins` property in `GlobalState`

Reply:
There is right here:
https://github.com/mattermost/mattermost-webapp/blob/eafb2f4b394f1d1788f1038b3ea19f0da32b6805/types/store/index.ts#L17

It links to another file if that helps.